core/src/main/res/values/dimens.xml

```diff
@@ -8,14 +8,16 @@
     <dimen name="text_size_navdrawer">16sp</dimen>
     <dimen name="text_size_medium">18sp</dimen>
     <dimen name="text_size_large">22sp</dimen>
+    <dimen name="text_size_semi_medium">18sp</dimen>
     <dimen name="thumbnail_length_itemlist">56dp</dimen>
     <dimen name="thumbnail_length_queue_item">56dp</dimen>
-    <dimen name="thumbnail_length_onlinefeedview">100dp</dimen>
+    <dimen name="newthumbnail_length_onlinefeedview">84dp</dimen>
     <dimen name="feeditemlist_header_height">132dp</dimen>
     <dimen name="thumbnail_length_navlist">40dp</dimen>
     <dimen name="listitem_iconwithtext_height">48dp</dimen>
     <dimen name="listitem_iconwithtext_textleftpadding">16dp</dimen>
-
+    <dimen name="widgetIconsize">14dp</dimen>
```

Comment:
Widget on Android could be misleading when referring to something like buttons. Given that this is used only for the "close" button, I would just specify its size manually without a dimen